SConstruct

```diff
@@ -290,6 +290,7 @@ for key in ["CFLAGS", "CXXFLAGS", "CCFLAGS"]:
 # Added by ourselves.
 extra_defines = [
     "WINDOWS_NO_FUTEX",
+    "NDEBUG",
```

Comment:
Makes sense for release builds, but probably should be only conditionally set it if `env["optimize"] != "debug"`, in case we need a build with enabled validation.

```python
if env["optimize"] != "debug":
    extra_defines += ["NDEBUG"]
```